source/docs/articles/wordpress/site-networks/getting-started.md

```diff
@@ -0,0 +1,90 @@
+
+### Create a Network with WP-CLI (Recommended)
+
+If you’re comfortable with the command line, Terminus and WP-CLI let you convert an existing WordPress environment to Multisite in one command: `wp core multisite-convert`
+
+Before you begin, make sure that:
+
+- Your WordPress environment is functioning as expected.
+- Terminus is configured locally, and you’re authenticated with Pantheon.
+
+Once you’ve confirmed these prerequisites, you can enable WordPress Multisite with one command.
+
+Subdirectory style:
+```bash
+terminus wp core multisite-convert --site=<pantheon-site> --env=dev
+```
+
+Subdomains style:
+```bash
+terminus wp core multisite-convert --site=<pantheon-site> --env=dev --subdomains
+```
+
+Optionally specify --title to use a custom title for your Multisite Network:
+
+```bash
+terminus wp core multisite-convert --site=<pantheon-site> --env=dev --title=”My Awesome Multisite Network”
+```
+
+For full compatibility with Pantheon, you’ll need to update DOMAIN_CURRENT_SITE to be set conditionally based on environment. Here is an example:
+
+https://gist.github.com/danielbachhuber/69c44664d4d63a6e19db
+
+### Caveats for Creating the Network Via the WordPress Dashboard
+
+WordPress Site Networks can also be created from the WordPress admin by following the [instructions at the codex](http://codex.wordpress.org/Create_A_Network).
+
+Once you’ve created the network, you’ll be taken to a new page:
+
+![](https://dl.dropboxusercontent.com/s/gu89aelpj4jqyin/2015-11-10%20at%209.52%20AM.png?dl=0)
+
+Copy the first block and add it to your site’s `wp-config.php` file.
+
+Add the constants to your wp-config right below where you added the WP_ALLOW_MULTISITE constant.
+
+Instead of defining DOMAIN_CURRENT_SITE explicitly, you’ll want to define it conditionally based on environment. Here is an example:
+
+https://gist.github.com/danielbachhuber/69c44664d4d63a6e19db
+
+Ignore the second block. Pantheon containers use Nginx + PHP-FPM, not Apache, and `.htaccess` files have no effect.
+
+Once you log back in to WordPress, pat yourself on the back — you’ve completed the Multisite installation process.
+
+## Develop the Site Network
+
+Now that you’ve made it through the installation process, congratulations on setting up your first WordPress Multisite environment. You are on your way to glory!
+
+When logged in to the WordPress Dashboard, you may notice a new “My Sites” menu item in the Toolbar. You can create your first site in the Network Admin:
+
+![](https://dl.dropboxusercontent.com/s/pxzxwfsvrt2xo5x/2015-11-14%20at%2011.05%20AM.png?dl=0)
+
+<div class="alert alert-info" role="alert">
+<h4>Note</h4>
+If you’re using the subdomain feature of WordPress Multisite, add the domain for each site you create to the Pantheon Dashboard (as well as configure requisite DNS settings) in order for the site to be publicly accessible. </div>
```

Comment:
We may want to link to the Domains doc here, and provide clarifying information on requirements for configuring multiple subdomains. Should those subdomains be added on all environments? Consider ditching the note style and creating a subsection for DNS configuration?